### PR TITLE
add (DCA,Pt,Mass) histogram to analyser - switch to histogram registry

### PR DESCRIPTION
**Pull request by aimeric-landou**

### CODE REVIEW

**Review by iarsene — APPROVED** [review]

**Comment by iarsene** [comment]
Hi @aimeric-landou,
I approved, but before merging, could you double check the output of the fullCI and macOS tests which fail? They seem unrelated to your commit, but please have a look for the macOS one.
Cheers,
Ionut

**Comment by aimeric-landou** [comment]
@iarsene the macOS went without issue ; was waiting for the fullCI to finish but the PR has just been merged
The fullCI test error message doesn't seem to point towards this commit.

**Pull request merged**
